core/json/node.go

```diff
@@ -8,7 +13,7 @@ type InfoNodeResult struct {
 	GraphState      GetGraphStateResult `json:"graphstate"`
 	TimeOffset      int64   `json:"timeoffset"`
 	Connections     int32   `json:"connections"`
-	Difficulty      float64 `json:"difficulty"`
+	PowDiff         PowDiff `json:"pow_diff"`
```

Comment:
please keep using `camel-case` see the example from https://golang.org/pkg/encoding/json/